utils/checks.py

```diff
@@ -95,3 +95,25 @@ def predicate(ctx):
         )
 
     return commands.check(predicate)
+
+
+def feature_flag(flag_name, use_ddb_user=False, default=False):
```

Comment:
added a `@feature_flag(flag)` decorator to easily control whether or not a command is enabled based on a feature flag